frontend/src/app/events/components/search/search.component.ts

```diff
@@ -55,9 +63,29 @@ export class SearchComponent implements OnInit {
     alert('Searching between times: ' + this.fromTime.value+ ' and ' + this.toTime.value);
   }
   search(){
+    this.loading = true;
```

Comment:
For loading spinner small changes are added,  to finalize loading by covering all pages in the future.